src/main/java/com/simanavets/tenderflexbackend/entity/Contract.java

```diff
@@ -0,0 +1,27 @@
+package com.simanavets.tenderflexbackend.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class Contract {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @ManyToOne
+    @JoinColumn(name = "offer_id")
+    private Offer offer;
```

Comment:
Contract can have relations with other entities just having it's ID. So all you need here is just ID's of the offer and the tender. And, i recommend use bidderId and contractorId as well, only because it will be easier to fetch it from DB in future